__tests__/app.test.js

```diff
@@ -40,8 +41,65 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of all article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.body).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with the requested article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.author).toBe("butter_bridge");
+        expect(body.article.title).toBe("Living in the shadow of a great man");
+        expect(body.article.article_id).toBe(1);
+        expect(body.article.body).toBe("I find this existence challenging");
+        expect(body.article.topic).toBe("mitch");
+        expect(body.article.created_at).toBe("2020-07-09T20:11:00.000Z");
+        expect(body.article.votes).toBe(100);
+        expect(body.article.article_img_url).toBe(
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
```

Comment:
Good work checking what the actual article looks like rather than the types of data, at the very least we should check the article_id is 1!